engine.py

```diff
@@ -130,6 +130,7 @@ def execute_command(self, cmd_key, args):
             # start up our QApp now, if none is already running
             qt_application = None
             if not QtGui.qApp:
+                QtGui.QApplication.setLibraryPaths([])
```

Comment:
Ah, I see where you are coming from now with this fix. I'm a bit worried about what unsetting this might do for some clients. Maybe its legitimate at some studios to set the QT_PLUGINS_PATH and then run the tank command. This would break those people. Let me come back to you.